src/components/form/EncounterForm.jsx

```diff
@@ -155,27 +151,21 @@ const mapStateToProps = (state, props) => {
         formInstanceId: props.formInstanceId,
         patient: props.patient,
         encounter: props.encounter,
-        encounterRole: props.encounterRole ? props.encounterRole : null,
+        encounterRole: props.encounterRole || props.encounterRole || null,
         encounterType: props.encounterType,
-        location: props.location ? props.location :
-          sessionLocation ? sessionLocation : null,
+        location: props.location || sessionLocation || null,
```

Comment:
this uses the session location variable that you removed above?

Reply:
I put it back